data/INRA/2016_harvest_data.ttl

```diff
@@ -0,0 +1,53 @@
+ <data/winemaking/PechRouge/Harvest/2016> a qb:DataSet; 
+  rdfs:label "Data about Harvest done on Pech Rouge in 2016"; 
+  qb:structure <DSD/harvestWeight/2016> . 
+ 
+<DSD/harvestWeight/2016> a qb:DataStructureDefinition; 
+  qb:component [qb:dimension bdg:plot                                                              ]; 
+  qb:component [qb:dimension bdg:variety                                                           ]; 
+  qb:component [qb:dimension bdg:date                                                              ]; 
+  qb:component [qb:dimension bdg:humanSensor                                                       ]; 
+  qb:component [qb:measure   bdg:grapeClusterWeight    ; qb:componentAttachment qb:DataSet         ]; 
+  qb:component [qb:attribute sosa:hasFeatureOfInterest ; qb:componentAttachment qb:MeasureProperty ]; 
+  qb:component [qb:attribute sdmx-attribute:unitMeasure; qb:componentAttachment qb:MeasureProperty ] . 
+ 
+<variety/Chardonnay> a bdg:Variety; 
+rdfs:label"Chardonnay"^^xsd:string . 
+ 
+<variety/Muscat-Petit-Grain> a bdg:Variety; 
+rdfs:label"Muscat-Petit-Grain"^^xsd:string . 
+
+<INRA/agent/fabien.robert> a bdg:humanSensor, ofpe:Operator, foaf:Person;
+  foaf:familyName "robert"^^xsd:string;
+  foaf:firstName "fabien"^^xsd:string .
+
+<INRA/agent/jerome.cerutti> a bdg:humanSensor, ofpe:Operator, foaf:Person;
+  foaf:familyName "cerutti"^^xsd:string;
+  foaf:firstName "jerome"^^xsd:string .
+
+<data/winemaking/PechRouge/harvestWeight/2016/51507e4d-d790-4406-850e-a67b0cc31839> a qb:Observation;
```

Comment:
As an observation is functionally defined by the dimensions they could be  used for making the uris. 
As a rule it is always good to avoid uuids where you can 

<data/winemaking/harvestWeight/PechRouge/81/81-CHARDONNAY/2016-08-24/Chardonnay/fabien.robert> a qb:Observation;

Reply:
I agree with you, I will procceed like this.

Reply:
The commit ca20410e9030b146b64be8384ff06b0b4eafffdd correct this issue.